semconv/v1.34.0/error_type.go

```diff
@@ -0,0 +1,31 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package semconv // import "go.opentelemetry.io/otel/semconv/v1.34.0"
+
+import (
+	"fmt"
+	"reflect"
+
+	"go.opentelemetry.io/otel/attribute"
+)
+
+// ErrorType returns an [attribute.KeyValue] identifying the error type of err.
+func ErrorType(err error) attribute.KeyValue {
```

Comment:
The coverage miss here is because we remove any code coverage result from semconv packages.
See https://github.com/open-telemetry/opentelemetry-go/blob/main/Makefile#L181

That makes sense, as the code is generated and we don't want the lack of coverage to crash our coverage report.